stepactions/fips-operator-check-step-action/0.2/fips-operator-check-step-action.yaml

```diff
@@ -0,0 +1,253 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: StepAction
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.2"
```

Comment:
@zxiong do we need a new stepaction version? IIUC, the new parameter isn't mandatory and wouldn't require a migration. Can we make this change in the existing stepaction?

Reply:
I'm fine with a new version too if you feel strongly about it. Also, please keep in mind we might have to bump the resources for the step using this stepAction in the FIPS tasks as we are running parallel scans.

Reply:
> @zxiong do we need a new stepaction version? IIUC, the new parameter isn't mandatory and wouldn't require a migration. Can we make this change in the existing stepaction?

Good point, we don’t need to bump the version.

> I'm fine with a new version too if you feel strongly about it. Also, please keep in mind we might have to bump the resources for the step using this stepAction in the FIPS tasks as we are running parallel scans.

Yes, a follow-up PR will address that once this is merged. Thanks for pointing it out.